### DISCUSSION TITLE
Welcome to rescript-chakra Discussions!

### DISCUSSION THREAD

**Discussion by r17x** [rescript-ui/rescript-chakra#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pewulfman** [comment]
Thanks mate, I am fairly new to front-end design and Rescript (OCaml dev). We used Bootstrap but didn't liked it, I want to change our UI lib. I just found out about Chakra and I am very happy to see that there are already some rescript bindings :).
Will test it and give you some feedbacks :)

**Reply by r17x** [reply]
This project has NOT been maintain for a long time, but I am ready to take care of it again.

I'm eagerly awaiting your feedback
